lua/render-markdown/init.lua

```diff
@@ -128,8 +129,9 @@ function M.setup(opts)
             latex = '@markup.math',
             quote = '@markup.quote',
         },
+        enabled = false,
     }
-    state.enabled = true
+    state.enabled = opts and opts.enabled or default_config.enabled
```

Comment:
I don't like the idea of de-syncing the `UserConfig` and overall `Config`.

Let's call this new field something like `start_enabled` or `default_enabled`.

In `state.lua` add the mandatory parameter:

```lua
---@class Config
---@field public start_enabled boolean
```

In `init.lua` add the optional parameter:

```lua
---@class UserConfig
---@field public start_enabled? boolean
```

Set the default value:

```lua
---@type Config
local default_config = {
    start_enabled = true,
```

Update how `state.enabled` is initialized:

```lua
state.config = vim.tbl_deep_extend('force', default_config, opts or {})
state.enabled = state.config.start_enabled
```

Reply:
Ah, I see now. I think I understand why I couldn't get my changes to work at first now. I only made changes to `init.lua` and I missed `state.lua`.

Reply:
The change to `state.lua` as far as adding the field is just for type annotations, it shouldn't have any functional impacts.

Setting `state.enabled` in `init.lua` is the bit that enables the functionality. Unsure why this didn't seem to work locally.